### PR TITLE
Fix getting started links

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -143,15 +143,21 @@ Install KubeVirt, Flannel and Cluster Network Operator:
...
 # Check for latest version
-RELEASE=v0.16.0
-kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt.yaml
+
+# On other OS you might need to define it like
+export ="v0.18.0"
+# On Linux you can obtain it using 'curl' via:
+export KUBEVIRT_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt/releases | grep tag_name | grep -v -- - | sort -V | tail -1 | awk -F':' '{print $2}' | sed 's/,//' | xargs)
+
+kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-operator.yaml
```
Comment (masayag) [inline docs/getting-started.md:152]: Add also the creation of the kubevirt-cr in order to deploy kubevirt resources by the operator.
```
kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml
```

Comment (masayag) [reply docs/getting-started.md:152]: it would probably a good decision to wait till kubevirt pods are available before moving to next step.

Comment (shiramax) [reply docs/getting-started.md:152]: @masayag done

File: docs/getting-started.md [hunk 2]
```diff
@@ -143,15 +143,24 @@ Install KubeVirt, Flannel and Cluster Network Operator:
 kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml
 
 # Check for latest version
-RELEASE=v0.16.0
-kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt.yaml
 
-kubectl apply -f https://raw.githubusercontent.com/kubevirt/ovs-cni/master/examples/kubernetes-ovs-cni.yml
+# On other OS you might need to define it like
+export KUBEVIRT_VERSION="v0.18.0"
```
Comment (masayag) [inline docs/getting-started.md:148]: current version is >> v0.30.0 - would you like to update it to more advanced one ?
Is it based on the recent work to install kubevirt on top of cento-8 ?

**Review by masayag — APPROVED** [review]

**Pull request merged**